src/lib/ActionPrototype.js

```diff
@@ -11,55 +11,50 @@ var ActionPrototype = {
     GOT_MSG: null
   }),
   originalActions: {},
-  stubForTests: function() {
+  stubForTests() {
```

Comment:
The changes in this file are just style cleanups (e.g., no parens around single arguments passed to arrow functions)